kkm/kkm_main.c

```diff
@@ -897,17 +897,6 @@ static void kkm_check_cpu_support(void)
 		printk(KERN_INFO "kkm_init: using X86_FEATURE_XSAVES.\n");
 	}
 
-#if (LINUX_VERSION_CODE < KERNEL_VERSION(5, 16, 0))
```

Comment:
We see the issue on 5.15. If I read this correctly this will not help there

Reply:
I deleted this code. No check any more.